app/models/forms/escort.rb

```diff
@@ -0,0 +1,7 @@
+module Forms
+  class Escort < Forms::Base
```

Comment:
Is there any specific need to inherit from `Forms::Base` ? If not, I would suggest to just have the form class to use the `ActiveModel::Validations` mixin instead.